src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidDependenciesTests.cs

```diff
@@ -95,7 +95,7 @@ public void GetDependencyWhenBuildToolsAreMissingTest ()
 				builder.Target = "GetAndroidDependencies";
 				Assert.True (builder.Build (proj, parameters: parameters),
 					string.Format ("First Build should have succeeded"));
-				int apiLevel = Builder.UseDotNet ? AndroidSdkResolver.GetMaxInstalledPlatform () : 26;
+				int apiLevel = Builder.UseDotNet ? 31 : 26;
```

Comment:
Should this "31" value come from "elsewhere"?  Perhaps using the `$(AndroidTargetDotNetApiLevel)` / `$(AndroidDefaultTargetDotnetApiLevel)` property?